flang/lib/Lower/IntrinsicCall.cpp

```diff
@@ -83,6 +84,10 @@ enum class ExtremumBehavior {
   // possible to implement it without some target dependent runtime.
 };
 
+// FIXME: consider merging IntrinsicLibrary class and the
+// InstrinsicCallOpsHelper class. They serve the same purpose and the methods
+// here can just be private to the helper.
+
```

Comment:
That's possible, but `IntrinsicLibrary` structure is tied to many implementation choices that we may want to revisit more frequently than the public interface, what is the benefit of exposing the implementation details in the public interface ?

Reply:
I left it because I wanted to see what you thought.

I don't think the difference is all that great. It's more stylistic, I'd say, and I don't have a strong argument one way or the other.